schemas/ispyb/stored_programs/sp_retrieve_apertures_using_size.sql

```diff
@@ -0,0 +1,13 @@
+DELIMITER ;;
+CREATE OR REPLACE DEFINER= `ispyb_root`@`%` PROCEDURE `retrieve_apertures_using_size`(p_sizeX double)
```

Comment:
I'm curious if this query is currently used for anything useful - is this call justified? What happens to the result?

Reply:
This is apparently used by the createDnaDataCollection method from gda.px.util.ExtendedCollectRequestProcessor to find a matching aperture Id for a given aperture size as a fallback if it doesn't find an aperture position in some beam profile. I do also think it's a strange query. I can't really comment on whether this method is still used/useful.

Reply:
I think we're waiting for Paul to figure out whether this is useful, but I'm happy to just approve this one for now.